server/metadata/access_token.go

```diff
@@ -0,0 +1,8 @@
+package metadata
```

Comment:
do we need it here? should we move it to server package?

Reply:
you mean next to main.go?  https://github.com/tigrisdata/tigris/tree/main/server

Reply:
Sorry i meant in service package